release-notes-2.1.1.md

```diff
@@ -0,0 +1,8 @@
+#### Version Number
+${version-number}
+
+#### New Features
```

Comment:
Don't call this a new feature or else we should be updating to `2.2.0`.

Reply:
Done